src/entities/ring-entity.ts

```diff
@@ -280,4 +288,14 @@ export class Ring {
     eager: true,
   })
   public statusOfRing: StatusOfRing;
+
+  @AfterLoad()
```

Comment:
I think that's a waste. The database stores this type as a numerical one.

Reply:
Fixed. Used tranformer instead